js/pages/cohort-definitions/cohort-definition-manager.js

```diff
@@ -459,7 +459,7 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 			
 			this.cohortDefinitionLink = ko.computed(() => {
 				if (this.model.currentCohortDefinition()) {
-					return this.config.api.url + "/cohortdefinition/" + this.model.currentCohortDefinition().id();
+					return this.config.api.url + "cohortdefinition/" + this.model.currentCohortDefinition().id();
```

Comment:
may be add common-utils function that combines url from set of strings with checking whether each part ends with slash or not?
I believe this would be useful